onnxruntime/core/providers/openvino/ov_versions/data_ops.cc

```diff
@@ -0,0 +1,1047 @@
+    {V_2020_4,{"CPU"},"Cos"},
+    {V_2020_4,{"CPU"},"Cosh"},
+    {V_2020_4,{"All"},"DepthToSpace"},
+    {V_2020_4,{"All"},"Div"},
+    {V_2020_4,{"All"},"Dropout"},
+    {V_2020_4,{"All"},"Elu"},
+    {V_2020_4,{"All"},"Equal"},
+    {V_2020_4,{"All"},"Erf"},
+    {V_2020_4,{"All"},"Exp"},
+    {V_2021_1,{"MYRIAD"},"Expand"},
+    {V_2020_4,{"All"},"Flatten"},
+    {V_2020_4,{"All"},"Floor"},
+    {V_2020_4,{"All"},"Gather"},
+    {V_2021_2,{"MYRIAD"},"GatherND"},
+    {V_2020_4,{"All"},"Gemm"},
+    {V_2020_4,{"All"},"GlobalAveragePool"},
+    {V_2020_4,{"CPU", "GPU"},"GlobalLpPool"},
+    {V_2020_4,{"All"},"Greater"},
+    {V_2020_4,{"All"},"Identity"},
+    {V_2020_4,{"All"},"InstanceNormalization"},
+    {V_2020_4,{"CPU", "GPU"},"HardSigmoid"},
+    {V_2020_4,{"All"},"LeakyRelu"},
+    {V_2020_4,{"All"},"Less"},
+    {V_2020_4,{"All"},"Log"},
+    {V_2021_2,{"MYRIAD"},"Loop"},
+    {V_2020_4,{"All"},"LRN"},
+    {V_2020_4,{"All"},"LSTM"},
+    {V_2020_4,{"All"},"MatMul"},
+    {V_2020_4,{"All"},"Max"},
+    {V_2020_4,{"All"},"MaxPool"},
+    {V_2020_4,{"All"},"Mean"},
+    {V_2020_4,{"All"},"Min"},
+    {V_2020_4,{"All"},"Mul"},
+    {V_2020_4,{"All"},"Neg"},
+    {V_2021_1,{"All"},"NonMaxSuppression"},
+    {V_2021_1,{"CPU", "MYRIAD"},"NonZero"},
+    {V_2021_1,{"All"},"Not"},
+    {V_2020_4,{"CPU", "GPU"},"Not"},
+    {V_2020_4,{"All"},"OneHot"},
+    {V_2020_4,{"All"},"Pad"},
+    {V_2020_4,{"All"},"Pow"},
+    {V_2020_4,{"All"},"PRelu"},
+    {V_2021_2,{"MYRIAD"},"Range"},
+    {V_2020_4,{"All"},"Reciprocal"},
+    {V_2020_4,{"CPU", "MYRIAD"},"ReduceLogSum"},
+    {V_2020_4,{"All"},"ReduceMax"},
+    {V_2020_4,{"All"},"ReduceMean"},
+    {V_2020_4,{"All"},"ReduceMin"},
+    {V_2020_4,{"CPU"},"ReduceProd"},
+    {V_2020_4,{"All"},"ReduceSum"},
+    {V_2020_4,{"CPU", "MYRIAD"},"ReduceSumSquare"},
+    {V_2020_4,{"All"},"Relu"},
+    {V_2020_4,{"CPU"},"Resize"},
+    {V_2020_4,{"All"},"Reshape"},
+    {V_2021_1,{"All"},"RoiAlign"},
+    {V_2021_2,{"MYRIAD"},"Round"},
+    {V_2021_1,{"MYRIAD"},"Scatter"},
+    {V_2021_2,{"MYRIAD"},"ScatterElements"},
+    {V_2020_4,{"CPU", "GPU"},"Selu"},
+    {V_2020_4,{"All"},"Shape"},
+    {V_2020_4,{"All"},"Sigmoid"},
+    {V_2020_4,{"CPU"},"Sign"},
+    {V_2020_4,{"CPU"},"Sign"},
+    {V_2020_4,{"CPU"},"Sinh"},
+    {V_2020_4,{"MYRIAD"},"SinFloat"},
+    {V_2020_4,{"All"},"Slice"},
+    {V_2020_4,{"All"},"Softmax"},
+    {V_2020_4,{"All"},"SpaceToDepth"},
+    {V_2020_4,{"All"},"Split"},
+    {V_2020_4,{"All"},"Sqrt"},
+    {V_2020_4,{"All"},"Squeeze"},
+    {V_2020_4,{"CPU"},"Softsign"},
+    {V_2020_4,{"All"},"Sub"},
+    {V_2020_4,{"All"},"Sum"},
+    {V_2020_4,{"CPU", "GPU"},"Tan"},
+    {V_2020_4,{"All"},"Tanh"},
+    {V_2021_2,{"MYRIAD"},"Tile"},
+    {V_2020_4,{"All"},"Transpose"},
+    {V_2020_4,{"All"},"TopK"},
+    {V_2020_4,{"All"},"Unsqueeze"},
+    {V_2021_1,{"CPU", "GPU"},"Upsample"},
+    {V_2021_2,{"MYRIAD"},"Where"},
+    
+};
+
+void Capability::populate_types_supported() {
```

Comment:
How will this function handle a new data type that would be supported with new version. say, we are supporting FP16 as Input precision on GPU and VPU only from 2021.1 Release. we might have to add int8 for keembay support from future releases. I think this should also be handled using  a multimap sort of thing (version, data_type_supported).

Reply:
I have added version id support